0~~~~                            ~~ Monitor: 8e231814-55fc-4321-83bb-f30a5d81ffc2
   2017-10-10 09h59.44.0780374   > Initializing TextFile handler (MaxCountPerFile = 20000).
                   +00.0019996   < 
-                  +00.0819981 F [Microsoft.AspNetCore.Server.Kestrel] Unable to start Kestrel.
                                    ┌──────────────────────────■ Exception: IOException ■──────────────────────────
                                    | Message:    Failed to bind to address http://127.0.0.1:4324: address already in use.
                                    | Stack:         à Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindEndpointAsync>d__5.MoveNext()
                                    |             --- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
                                    |                à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
                                    |                à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
                                    |                à Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindLocalhostAsync>d__6.MoveNext()
                                    |             --- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
                                    |                à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
                                    |                à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
                                    |                à Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindAddressAsync>d__7.MoveNext()
                                    |             --- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
                                    |                à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
                                    |                à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
                                    |                à Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.AddressesStrategy.<BindAsync>d__2.MoveNext()
                                    |             --- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
                                    |                à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
                                    |                à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
                                    |                à Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindAsync>d__0.MoveNext()
                                    |             --- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
                                    |                à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
                                    |                à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
                                    |                à Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServer.<StartAsync>d__21`1.MoveNext()
                                    |  ┌──────────────────────────■ [Inner Exception] ■──────────────────────────
                                    |  |  ┌──────────────────────────■ Exception: AddressInUseException ■──────────────────────────
                                    |  |  | Message:    Error -4091 EADDRINUSE address already in use
                                    |  |  | Stack:         à Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.LibuvTransport.<BindAsync>d__20.MoveNext()
                                    |  |  |             --- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
                                    |  |  |                à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
                                    |  |  |                à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
                                    |  |  |                à Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServer.<>c__DisplayClass21_0`1.<<StartAsync>g__OnBind0>d.MoveNext()
                                    |  |  |             --- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
                                    |  |  |                à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
                                    |  |  |                à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
                                    |  |  |                à Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindEndpointAsync>d__5.MoveNext()
                                    |  |  |  ┌──────────────────────────■ [Inner Exception] ■──────────────────────────
                                    |  |  |  |  ┌──────────────────────────■ Exception: UvException ■──────────────────────────
                                    |  |  |  |  | Message:    Error -4091 EADDRINUSE address already in use
                                    |  |  |  |  | Stack:         à Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.LibuvFunctions.tcp_getsockname(UvTcpHandle handle, SockAddr& addr, Int32& namelen)
                                    |  |  |  |  |                à Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.UvTcpHandle.GetSockIPEndPoint()
                                    |  |  |  |  |                à Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Listener.ListenTcp(Boolean useFileHandle)
                                    |  |  |  |  |                à Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Listener.CreateListenSocket()
                                    |  |  |  |  |                à Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Listener.<>c.<StartAsync>b__8_0(Listener listener)
                                    |  |  |  |  |                à Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.LibuvThread.CallbackAdapter`1.<>c.<.cctor>b__3_1(Object callback, Object state)
                                    |  |  |  |  |                à Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.LibuvThread.DoPostWork()
                                    |  |  |  |  |             --- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
                                    |  |  |  |  |                à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
                                    |  |  |  |  |                à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
                                    |  |  |  |  |                à Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.ListenerPrimary.<StartAsync>d__15.MoveNext()
                                    |  |  |  |  |             --- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
                                    |  |  |  |  |                à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
                                    |  |  |  |  |                à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
                                    |  |  |  |  |                à System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
                                    |  |  |  |  |                à Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.LibuvTransport.<BindAsync>d__20.MoveNext()
                                    |  |  |  |  └─────────────────────────────────────────────────────────────────────────
                                    |  |  |  └─────────────────────────────────────────────────────────────────────────
                                    |  |  └─────────────────────────────────────────────────────────────────────────
                                    |  └─────────────────────────────────────────────────────────────────────────
                                    └─────────────────────────────────────────────────────────────────────────
